src/components/DisciplineGroup.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react'
+import { cragFiltersStore } from '../js/stores'
+import { FilterToggleButton } from './ui/Button'
+
+const DisciplineGroup = ({ onChange }): JSX.Element => {
+  const { trad, sport, tr, boulder } = cragFiltersStore.useStore()
```

Comment:
does it need to be dynamic?
```javascript
const items = { trad, sport, tr, boulder } // these are boolean values.

const [pillDetails, setPillDetails] = useStore(Object.values(items));

return (
  <div className='flex space-x-2'>
    Object.keys(items).map( (pill, index) => 
        <FilterToggleButton
            key={index}
            selected={pillDetails[index]}
            label={pill} onClick={() => {
            let tempPillDetails = pillDetails
            tempPillDetails[index] = pillDetails[index]
            setPillDetails([...tempPillDetails])
            onChange(pill)
        }}
      />
   ) 
)
```
sorry if this is formatted bad

wrote this up in the really small github window, probably should have messed with the browser styles or did in an ide.

<img width="738" alt="Screen Shot 2022-04-01 at 11 47 43 AM" src="https://user-images.githubusercontent.com/11463275/161298005-bfafa0fb-7677-43a6-9824-9c3ff41f1a74.png">